client/app/components/composites/AvatarDropdown/AvatarDropdown.js

```diff
@@ -37,13 +64,16 @@ class AvatarDropdown extends Component {
   }
```

Comment:
The timeouts should be cleared also in `componentWillUnmount` to avoid changing the state after the component is unmounted.